src/caustics/light/func/point_source.py

```diff
@@ -0,0 +1,17 @@
+from torch import where
+
+from ...utils import translate_rotate
+
+
+def brightness_point(x0, y0, theta_s, Ie, x, y, gamma=1.0, s=0.0):
```

Comment:
I don't think a default value is needed for `gamma` and if there is a default value, I would set it to `0.0` for no limb darkening to make the default behaviour more like one would expect. Unless `1.0` is some special value for limb darkening.

Reply:
My mistake. I meant to set gamma to 0.0 for no limb darkening by default, which I think makes sense. I believe most users would assume no limb darkening as a starting point; it is often treated as a higher-order effect and the coefficient can take a wide range of values.